### DISCUSSION TITLE
Welcome to telegram-auto_forwarder Discussions!

### DISCUSSION THREAD

**Discussion by Ayush7445** [Ayush7445/telegram-auto_forwarder#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by txpredictions** [comment]
Hi,
Is it. Possible to add remove mentions? If yes please share how to remove mentions in the message footer of forwarded message.
